core/control-plane/control-plane-aggregate-services/src/main/java/org/eclipse/edc/connector/controlplane/services/protocol/VersionProtocolServiceImpl.java

```diff
@@ -9,32 +9,27 @@
  *
  *  Contributors:
  *       Bayerische Motoren Werke Aktiengesellschaft (BMW AG) - initial API and implementation
+ *       Cofinity-X - unauthenticated DSP version endpoint
  *
  */
 
 package org.eclipse.edc.connector.controlplane.services.protocol;
 
-import org.eclipse.edc.connector.controlplane.services.spi.protocol.ProtocolTokenValidator;
 import org.eclipse.edc.connector.controlplane.services.spi.protocol.ProtocolVersionRegistry;
 import org.eclipse.edc.connector.controlplane.services.spi.protocol.ProtocolVersions;
 import org.eclipse.edc.connector.controlplane.services.spi.protocol.VersionProtocolService;
-import org.eclipse.edc.policy.context.request.spi.RequestVersionPolicyContext;
-import org.eclipse.edc.spi.iam.TokenRepresentation;
 import org.eclipse.edc.spi.result.ServiceResult;
 
 public class VersionProtocolServiceImpl implements VersionProtocolService {
 
     private final ProtocolVersionRegistry registry;
-    private final ProtocolTokenValidator tokenValidator;
 
-    public VersionProtocolServiceImpl(ProtocolVersionRegistry registry, ProtocolTokenValidator tokenValidator) {
+    public VersionProtocolServiceImpl(ProtocolVersionRegistry registry) {
         this.registry = registry;
-        this.tokenValidator = tokenValidator;
     }
 
     @Override
-    public ServiceResult<ProtocolVersions> getAll(TokenRepresentation tokenRepresentation) {
-        return tokenValidator.verify(tokenRepresentation, RequestVersionPolicyContext::new)
```

Comment:
is this `RequestVersionPolicyContext` not used anymore right? :axe:

Reply:
good point! other than registration of scopes/policy contexts doesn't seem to be used anywhere else. Since thematically it fits to this PR, I'll do so right away 🪓